roles/rancher/tasks/main.yml

```diff
@@ -470,11 +470,15 @@
   delay: 2
 
 - name: Add CloudVE Helm repo
-  shell: /usr/local/bin/helm repo add galaxyproject https://raw.githubusercontent.com/CloudVE/helm-charts/master/
+  shell: /usr/local/bin/helm repo add galaxyproject https://raw.githubusercontent.com/cloudve/helm-charts/master/
 
 - name: Update Helm repos
   shell: /usr/local/bin/helm repo update
 
+- name: Fallback to hostPath provisioner when no cloud provider is set
+  shell: /usr/local/bin/kubectl apply -f https://gist.githubusercontent.com/almahmoud/92186a6521cde97f3680a85cf5202bdc/raw/16d08cc4abc2b3f87ab02dec2dd479e4e9fdbd41/fallback-cloudman-boot-storageclass.yaml
```

Comment:
This should be checked into source control instead of being a gist, probably in cloudman-boot itself. Also, it may as well be baked into the image to reduce network comms/failures. Can we cleanly uninstall this if needed? If not, a better option might be to use a helm chart for this: https://hub.helm.sh/charts/rimusz/hostpath-provisioner

Reply:
The uninstall would be just kubectl delete -f [same-path] (presumably the baked-in file). We can use the helm chart if we think we want to customize this in the future, otherwise it seems like having the minimal version in a single file baked in would do the trick as a dev single-VM fallback?